_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,20 +24,26 @@ acknowledgments:
 
 This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the `content` attribute specified value includes the text `user-scalable`; or
+- the `content` attribute specified value includes the text `maximum-scale`.
+
+**Note:** The Applicability refers to string inclusion of the indicated strings in the value of the `content` attribute, as written.
 
 ## Expectation 1
 
 For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `no`.
 
+The [attribute value][] of the `content` attribute is a list of key-value pairs, obtained by parsing the specified value, then applying the [translations into `@viewport` descriptors][descriptor translation], notably [the translation for `user-scalable`](https://www.w3.org/TR/css-device-adapt-1/#user-scalable).
```

Comment:
I'm finding this has the potential to confuse people. By referencing the transplantation to @viewport blocks, it becomes harder to understand what really is checked. A specified attribute of `no` originates an attribute value of `no` which gets translated into `fixed` for the `user-zoom` descriptor. 

The descriptor will never have a value of `no`, and is not what the expectation looks at. Therefore, I'm not sure this sentence adds value to understanding the expectation. 

Same applies to Expectation 2.

Reply:
Good points.
* Expectation 2, and the way it is written ("does not have a value less than 2") is pretty much the reason why we need to refer to these translations, since we want, e.g. `maximum-scale=-1` to pass (Passed Example 3). I am not sure we can do without referring to the translation one way or another.
* Expectation 1 is more interesting since it seems to be yet another meaning of "have" 🙈
* Notably, there are no example to show whether we wanted to accept `user-scalable=0`. It is "specified" as `0`, but "computed" as `fixed` and therefore passes the rule (it does not have a value of `no` in any decent sense of the term because "translated into the same value as `no` is translated" is definitely not a decent sense for "have"…) but has the exact same effect as `user-scalable=no` which fails it 💥 

I think we messed up in the latest update and Expectation 1 should say "does not have a `user-scalable` property with a value of **`fixed`**." Plus keep the note about "have" meaning "after translation" (which would match Expectation 2)

Reply:
But from what I understand (please, correct me if I'm wrong), it's not the `user-scalable` property that gets the value fixed. It's the `user-zoom` descriptor that gets that value. So, I guess we have two choices:

1. We mention the `user-zoom` descriptor in the expectation
2. We update it to say that `user-scalable` property must not have a value of `no`, or in the range of -1 to 1, or an unknown value.

Reply:
Ah, yes, I see your point.
I think we should avoid rephrasing the translation, and that was a bit the goal of #1822 (no need to duplicate that and if CSS decides to change it, the rule should probably follow suit).

What about adding something like "…and using the value obtained for `user-zoom`" at the end of the sentence?

Reply:
That could work